js/src/Ice/AsyncResult.js

```diff
@@ -40,7 +40,7 @@ export class AsyncResult extends Promise {
     }
 
     markSent(done) {
-        DEV: console.assert((this._state & AsyncResult.Done) === 0);
+        console.assert((this._state & AsyncResult.Done) === 0);
```

Comment:
I removed this `DEV: ` labels, and use console.assert directly. Bundlers can be configure to ignore them, no need for a custom label.